### PR TITLE
Add scheduled job that refreshes vacancy facets daily

### PR DESCRIPTION
**Pull request by cesidio**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1433

### CODE REVIEW

File: spec/jobs/refresh_vacancy_facets_job_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  it 'queues the job' do
+    expect { job }.to change(ActiveJob::Base.queue_adapter.enqueued_jobs, :size).by(1)
+  end
+
+  it 'is in the refresh_vacancy_facets queue' do
+    expect(job.queue_name).to eq('refresh_vacancy_facets')
+  end
+
+  it 'invokes the libs to export users and approvers to big query' do
```
Comment (cpjmcquillan) [inline spec/jobs/refresh_vacancy_facets_job_spec.rb:16]: copy pasta 🍝

**Review by cpjmcquillan — APPROVED** [review]
small comment, but all good apart from that 👍

**Comment by csutter** [comment]
Looks good but I'll defer to the sharper eyes around here 👍 

One semi-related thing (probably for a separate docs cleanup PR) I've noticed as I'm getting set up and going through the README is that it talks about two timed/scheduled jobs when we've got a fair bunch more now.

File: spec/jobs/refresh_vacancy_facets_job_spec.rb (hunk 1 above)
Comment (cesidio) [reply spec/jobs/refresh_vacancy_facets_job_spec.rb]: yes too much pasta...

**Comment by cesidio** [comment]
> Looks good but I'll defer to the sharper eyes around here 👍
> 
> One semi-related thing (probably for a separate docs cleanup PR) I've noticed as I'm getting set up and going through the README is that it talks about two timed/scheduled jobs when we've got a fair bunch more now.

I believe that the two scheduled jobs that you refer to in the README are related to the Algolia section.

**Pull request merged**